Apps/WebClient/src/ClientApp/public/index.html

```diff
@@ -21,6 +21,12 @@
 
         <!-- Snowplow starts plowing - Standalone vE.2.14.0 -->
         <script type="text/javascript">
+            var sBrowser, sUsrAg = window.navigator.userAgent;
```

Comment:
official doc for this approach: https://developer.mozilla.org/en-US/docs/Web/API/Window/navigator
for info, after having a hard time to integrate modernizr to vue js, the above approach seems to be straight forward for this requirement.